src/main/webapp/app/view/speaker/QuestionStatisticChart.js

```diff
@@ -165,7 +164,36 @@ Ext.define('ARSnova.view.speaker.QuestionStatisticChart', {
 				handler: this.toggleCorrectHandler,
 				hidden: !hasCorrectAnswers() || this.questionObj.questionType === 'grid' ||
 					(ARSnova.app.userRole === ARSnova.app.USER_ROLE_STUDENT && !this.questionObj.showAnswer)
-			}]
+			}, {
+				xtype: 'button',
+				text: Messages.QUESTIONS_CSV_EXPORT_ANSWERS_BUTTON,
+				align: 'right',
+				handler: function () {
```

Comment:
When the export function is in the QuestionExport Controller, you can reuse it here to not repeat yourself.